.github/workflows/delivery-docker.yml

```diff
@@ -0,0 +1,27 @@
+name: delivery / docker
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  deliver-docker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Determine version
+        id: version
+        shell: bash
+        run: |
+          echo "::set-output name=VERSION::$(echo ${{ github.event.release.tag_name }} | cut -d "v" -f2)"
```

Comment:
I tried extensively to use `set-env`, and then consume it later in the file, but it didn't seem to work as an input to a task, regardless of what I tried. 

[This](https://github.community/t/how-to-pass-environment-variable-to-an-input/16277/2) seems to say that it should be done with output, though the solution is a bit dated. If you can get it working with `set-env`, let me know, I'm all ears!

Reply:
FWIW your implementation is what we are doing for the lifecycle image: https://github.com/buildpacks/lifecycle/blob/main/.github/workflows/lifecycle-image.yml